source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -125,6 +134,8 @@ class ExtProcLoggingInfo : public Envoy::StreamInfo::FilterState::Object {
   uint64_t bytes_sent_{0}, bytes_received_{0};
   Upstream::ClusterInfoConstSharedPtr cluster_info_;
   Upstream::HostDescriptionConstSharedPtr upstream_host_;
+  // The status details of the underlying HTTP/2 stream. Envoy gRPC only.
+  absl::optional<std::string> http_response_code_details_;
```

Comment:
What's the syntax difference between absl::nullopt vs an empty std::string here? 

might as well use std::string directly if none.  with optional you'd still pay the ram cost of a std::string already.

Reply:
Done.